### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by ivoflipse** [conda-forge/caffe-feedstock#3]
I was just wondering how hard it would be to add Windows support, since Microsoft seems to maintain a special branch for it: https://github.com/BVLC/caffe/tree/windows

But I'm not sure if all the requirements are available on the conda-forge or defaults channels

**Comment by jakirkham** [comment]
Yeah, we don't have all the Windows requirements. That's part of the problem.

That branch is interesting. I see a PR ref for it, but the link appears broken. Do you know what happened here?
